### PR TITLE
BAU — Use assertThrows in tests that assert exception messages

### PR DESCRIPTION
**Pull request by alexbishop1**
In JUnit 5, `assertThrows` returns the thrown exception so we can assert on it. This is nicer than using `try`/`catch`.

### CODE REVIEW

File: .secrets.baseline [hunk 1]
```diff
@@ -0,0 +1,75 @@
+{
+  "exclude": {
+    "files": null,
+    "lines": null
+  },
+  "generated_at": "2020-11-10T09:39:42Z",
```
Comment (adityapahuja) [inline .secrets.baseline:6]: Do we need `.secrets.baseline` file?

**Review by oswaldquek — APPROVED** [review]
oooh nice

**Review by adityapahuja — APPROVED** [review]
These changes make tests in CardTypeTest easier to read!

File: .secrets.baseline (hunk 1 above)
Comment (alexbishop1) [reply .secrets.baseline:6]: We do if we use https://github.com/alphagov/gds-pre-commit, which agreed to use after a key was committed that shouldn’t have been.

**Pull request merged**